src/aiida_vasp/parsers/vasp.py

```diff
@@ -54,6 +54,8 @@ class MissingFileError(ParserError):
 
 DEFAULT_EXCLUDED_QUANTITIES = (
     'energies',
+    'all_forces',
+    'all_stress',
```

Comment:
What are `all_forces` and `all_stress`? Are these stored in Array nodes or in misc?

Reply:
> What are all_forces and all_stress?

I found it in your initial PR comment! These are stored in `misc`?